src/accommodation/accommodation.controller.ts

```diff
@@ -111,43 +112,42 @@ export class AccommodationController {
   @ApiConsumes('multipart/form-data')
   @ApiBody({
     schema: {
-      type: 'object',
-      properties: {
-        file: {
-          type: 'string',
-          format: 'binary',
-          description: 'Image file (only .jpg, .jpeg, .png allowed), size < 10MB!',
+      type: 'array',
+      items: {
+        type: 'object',
+        properties: {
+          filename: { type: 'string' },
+          base64Image: { type: 'string' },
         },
       },
+      description: 'Images file (only .jpg, .jpeg, .png allowed), size < 10MB!',
```

Comment:
create please constant for max image size  and use it everywhere in the project/file